<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenamingCompatOptionsPage</class>
 <widget class="QWidget" name="RenamingCompatOptionsPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0,0">
   <item>
    <widget class="QCheckBox" name="ascii_filenames">
     <property name="text">
      <string>Replace non-ASCII characters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="windows_compatibility">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Windows compatibility</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_windows_compatibility_change">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Customize...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="windows_long_paths">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Allow paths longer than 259 characters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="replace_spaces_with_underscores">
     <property name="text">
      <string>Replace spaces with underscores</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_replace_dir_separator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Replace directory separators with:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replace_dir_separator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">_</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="example_selection_note">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ascii_filenames</tabstop>
  <tabstop>windows_long_paths</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>windows_compatibility</sender>
   <signal>toggled(bool)</signal>
   <receiver>windows_long_paths</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>windows_compatibility</sender>
   <signal>toggled(bool)</signal>
   <receiver>btn_windows_compatibility_change</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
